### PR TITLE
Add the ElaoStrap assets installation

### PR DESCRIPTION
**Pull request by supertanuki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+
+    /* import elaostrap css */
+    @import "elaostrap/assets/sass/style";
+    
+    /* import your own css */
+    @import "something/amazing";
+    
+Then do in your Vagrant VM :
+
+    $ gulp
```
Comment (benji07) [inline README.md]: `gulp install` instead of `gulp`

File: README.md [hunk 2]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+    
+            {% block javascripts %}
+                <script src="{{ asset('assets/js/web/main.js') }}"></script>
+            {% endblock %}
+        </body>
+    </html>
+
+If you choose to create a  WebBundle to store assets, here an example of file structure and some code :
+    
+Create the file WebBundle/Resources/assets/js/web/main.js :
```
Comment (benji07) [inline README.md]: AppBundle (to follow the new architecture of the symfony/symfony-standard

File: README.md [hunk 3]
```diff
@@ -50,3 +50,164 @@ FAQ
 ---
 
 Go to the [FAQ](https://github.com/Elao/symfony-standard/wiki/FAQ)
+
+Install ElaoStrap (optionnal)
```
Comment (JeremyJames) [inline README.md]: Optional

File: README.md [hunk 4]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+---
+
+Install our custom twitter bootstrap Symfony bundle : [ElaoStrap](https://github.com/Elao/ElaoThemeElaoStrapBundle)
```
Comment (JeremyJames) [inline README.md]: C'est pas vraiment un "custom twitter bootstrap symfony bundle', mais plutôt un "ElaoStrap theme bundle for Symfony"

File: README.md [hunk 5]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+    
+Update your /package.json. Here an example :
```
Comment (JeremyJames) [inline README.md]: here IS an exemple: | here's an exemple:
Il n'y a pas d'espace avant la ponctuation en anglais (? : ; !)

File: README.md [hunk 6]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+      "devDependencies": {
+          "gulp": "3.8.*",
+          "del": "1.1.*",
+          "elao-assets-gulp": "~0.1.16",
+          "napa": "1.1.*",
+          "browserify-shim": "3.8.*"
+      }
+    }
+    
+Update your /gulpfile.js, here an example :
```
Comment (JeremyJames) [inline README.md]: La même qu'en haut

File: README.md [hunk 7]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+        <body>
+            {% block body %}{% endblock %}
+    
+            {% block javascripts %}
+                <script src="{{ asset('assets/js/web/main.js') }}"></script>
+            {% endblock %}
+        </body>
+    </html>
+
+If your Bundle is AppBundle where you store your assets, here an example of files structure and some code :
```
Comment (JeremyJames) [inline README.md]: If the Bundle where you store your assets is AppBundle, here is an example of files structure and some code:

File: README.md [hunk 8]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+                }
+            }
+        }
+    });
+    
+Then do in your Vagrant VM :
+
+    $ npm install
+    
+An example how to call the assets in your app/Resources/views/base.html.twig :
```
Comment (JeremyJames) [inline README.md]: An example showing how to call

File: README.md [hunk 9]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+    
+            {% block javascripts %}
+                <script src="{{ asset('assets/js/web/main.js') }}"></script>
+            {% endblock %}
+        </body>
+    </html>
+
+If your Bundle is AppBundle where you store your assets, here an example of files structure and some code :
+    
+Create the file AppBundle/Resources/assets/js/web/main.js :
```
Comment (JeremyFagis) [inline README.md]: Pas besoin de mettre dans un sous dossier /web, c'était spécifique au projet.

File: README.md [hunk 10]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+        datepicker     = require('elaostrap.datepicker'),
+        select2        = require('select2');
+    
+    require('select2.fr');
+    
+    $(document).ready(function(){
+        // your own code
+    });
+    
+Create the file AppBundle/Resources/assets/sass/web/main.scss :
```
Comment (JeremyFagis) [inline README.md]: Idem pour le sous dossier /web.

File: README.md [hunk 11]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+    <html>
+        <head>
+            <title>{% block title %}My title{% endblock %}</title>
+            <meta name="description" content="">
+            <meta charset="utf-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+            <meta name="format-detection" content="telephone=no">
+            {% block stylesheets %}
+                <link rel="stylesheet" href="{{ asset('assets/css/web/main.css') }}">
```
Comment (JeremyFagis) [inline README.md]: 'assets/css/web/main.css' -> 'assets/css/main.css'

File: README.md [hunk 12]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+      "napa": {},
+      "dependencies": {
+          "bootstrap-sass": "3.3.1",
+          "elao-form.js": "git://github.com/elao/form.js",
+          "elaostrap": "0.1.12",
+          "dropify": "0.0.6",
+          "fastclick": "1.0.3",
+          "jquery": "2.1.1",
+          "select2": "3.5.2-browserify",
+          "font-awesome": "4.3.0"
```
Comment (JeremyFagis) [inline README.md]: Pas indispensable d'intégrer font-awesome.

File: README.md [hunk 13]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+Update your /gulpfile.js, here an example :
+    
+    assets.config({
+        (...),
+        
+        assets: {
+            fonts: {
+                groups: {
+                    'elaostrap': {src: 'elaostrap/assets/fonts/**'},
+                    'font-awesome': {src: 'font-awesome/fonts/**', dest: 'font-awesome'},
```
Comment (JeremyFagis) [inline README.md]: Pas indispensable d'intégrer font-awesome.

**Comment by benji07** [comment]
I think we need to add this in the wiki (and note at the end of the readme, to keep the readme as short as possible)

File: README.md [hunk 14]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+
+Install ElaoStrap (optionnal)
+---
+
+Install our custom twitter bootstrap Symfony bundle : [ElaoStrap](https://github.com/Elao/ElaoThemeElaoStrapBundle)
+    
+Update your /package.json. Here an example :
+
+    {
+      "name": "project-name",
```
Comment (emerick42) [inline README.md]: You are using 2 spaces indents for the first level and the author object, but 4 spaces indents everywhere else. Maybe we can agree to use only 4 spaces indents?

File: README.md [hunk 15]
```diff
@@ -50,3 +50,164 @@ FAQ
...
+                groups: {
+                    'elaostrap': {src: 'elaostrap/assets/fonts/**'},
+                    'font-awesome': {src: 'font-awesome/fonts/**', dest: 'font-awesome'},
+                    'dropify': {src: 'dropify/src/fonts/**', dest: 'dropify'},
+                }
+            }
+        }
+    });
+    
+Then do in your Vagrant VM :
```
Comment (emerick42) [inline README.md]: No space before `:`

File: README.md (hunk 7 above)
Comment (emerick42) [inline README.md]: You can even say something like: "Here is an example of files structure and code, considering the bundle where you store your assets is called AppBundle:"

**Comment by supertanuki** [comment]
I move it to the wiki: https://github.com/Elao/symfony-standard/wiki/Install-ElaoStrap-(optional)

**Comment by benji07** [comment]
@supertanuki I just update your code to add syntax coloration

**Pull request closed without merging**